### PR TITLE
SMD-334: switch to paketo buildpack

### PR DESCRIPTION
**Pull request by gidsg**
### Change description
Switches deployed environments to use [packeto buildpack](https://paketo.io/)

N.B. the local Dockerfile is retained for local development only

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Have deployed on dev environment https://github.com/communitiesuk/funding-service-design-post-award-submit/actions/runs/6772020403/job/18403679668

### CODE REVIEW

File: project.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[ _ ]
+schema-version = "0.2"
```
Comment (cyrusdobbs) [inline project.toml:2]: What's schema version?

**Review by cyrusdobbs — COMMENTED** [review]
Should these paketo / asset building changes be accompanied with some changes to documentation?

**Comment by gidsg** [comment]
> Should these paketo / asset building changes be accompanied with some changes to documentation?

@cyrusdobbs Looking at it, I don't think the README is inaccurate but I could add something about how we run in production for informational purposes.

**Comment by cyrusdobbs** [comment]
> > Should these paketo / asset building changes be accompanied with some changes to documentation?
> 
> @cyrusdobbs Looking at it, I don't think the README is inaccurate but I could add something about how we run in production for informational purposes.


Happy for you to make a judgement on that - I'll approve this PR for now

**Review by cyrusdobbs — APPROVED** [review]
LGTM

**Pull request merged**